src/UI/Movies/FullMovieCollection.js

```diff
@@ -1,6 +1,7 @@
 var movieCollection = require('./MoviesCollection');
 
 var fullCollection = movieCollection.clone();
+fullCollection.reset();
```

Comment:
Im not sure I totally understand what this does but I dont see any problems with it.